FrontEnd/src/components/ProfileDetail/DetailedInfo/PhoneEmail.jsx

```diff
@@ -6,7 +6,9 @@ import { PropTypes } from 'prop-types';
 import classes from './PhoneEmail.module.css';
 import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
 import { toast } from 'react-toastify';
+import {Tooltip} from 'antd';
 
+const LengthEmail = 23;
```

Comment:
I think, you should rename this variable to upper case (LENGTH_EMAIL) since it is a constant and add a blank line after its definition.